robot/client/client.go

```diff
@@ -346,7 +346,10 @@ func (rc *RobotClient) connectWithLock(ctx context.Context) error {
 	if err := rc.conn.Close(); err != nil {
 		return err
 	}
-	conn, err := grpc.Dial(ctx, rc.address, rc.logger, rc.dialOptions...)
+
+	cancelCtx, stop := context.WithTimeout(ctx, time.Second)
```

Comment:
Oops, need to revert this and see if there's a better option for speeding up the test.